src/libraries/System.ServiceProcess.ServiceController/src/System.ServiceProcess.ServiceController.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum);netstandard2.0-windows;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
-    <NoWarn>$(NoWarn);CA2249</NoWarn>
+    <NoWarn>$(NoWarn);CA1066;CA2249</NoWarn>
```

Comment:
@ericstj, I could use your help on this one.  I tried to make SessionChangeDescription implement `IEquatable<SessionChangeDescription>`, but I couldn't come up with the magic incantation to make all the builds happy with API compat and not-supported builds, so I just skipped it.

Reply:
You should be able to just add that API to [System.ServiceProcess.ServiceController.netcoreapp.cs](https://github.com/dotnet/runtime/blob/26b566ec87e5fb3b27d0511f1ed8819cd48344f2/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.netcoreapp.cs) and then condition the implementation appropriately.  I can try to suggest the appropriate change.

Reply:
Here's a suggestion: https://github.com/ericstj/runtime/commit/190ef10cc7575312fb16a9b0521bb6ca54950d98

Reply:
Thanks, @ericstj.  That seems to work.